### ISSUE TITLE
Image file shows up as featured image

### ISSUE THREAD

**Issue by norbertomoreau** [Automattic/WP-Job-Manager#598]
This image file shows up as featured image. Most logos uploaded are too small to show it as a banner. Looks terrible. Is there a work around that it only shows up as thumb in the listing header?

**Comment by mikejolley** [comment]
Templates/themes output featured images - this can be hidden with CSS. I'd need to see the page to see if there is a generic rule we can add to hide it.

**Comment by norbertomoreau** [comment]
Solved the topic by blanking out the wp-job-manager/templates/form-fields/_file-field.php and substitute the wp-job-manager/assets/images/company.png file with a website logo (showing the transparent logo on the top third of the square). Looks fine and the garbage logos of the companies won´t fill up my server. 

PS: regarding the other question with the login I found a solution. I just renamed the slug my-account -> "My Account" to myaccount leaving "My Account" in the header. Works well. 
(will paste this to the closed thread if you like, perhaps it helps others.)

I think you did y great job with this plugin. Keep on going!! Curious about the more! Will bookmark your page! (y)
